src/index.ts

```diff
@@ -10,11 +10,11 @@ function computePercentage(value: string, total: number) {
   return total * (percentage / 100)
 }
 
-type PaddingFormat =
+export type PaddingFormat =
```

Comment:
Are you exporting these types for a particular reason?

Reply:
Yes, since is a complex type I think could be useful to have